modules/swagger-codegen/src/main/resources/Java/enumClass.mustache

```diff
@@ -1,5 +1,6 @@
 public enum {{datatypeWithEnum}} {
-  {{#allowableValues}}{{#enumVars}}{{name}}("{{value}}"){{^-last}}, {{/-last}}{{#-last}};{{/-last}}{{/enumVars}}{{/allowableValues}}
+  {{#allowableValues}}{{#enumVars}}{{name}}("{{value}}"){{^-last}},
+  {{/-last}}{{#-last}};{{/-last}}{{/enumVars}}{{/allowableValues}}
```

Comment:
You should do the same change in the JavaInflector, JavaJaxRS and JavaSpringMVC templates (there the code is in model.mustache).